### PR TITLE
Fix jruby builds

### PR DESCRIPTION
**Pull request by smudge**
/domain @asullivan210 @samandmoore @aburgel 
/no-platform

Pinning SQL adapter down to a version that should work.

### CODE REVIEW

**Comment by smudge** [comment]
I lied -- one build still broken. Fixing that now.

**Comment by aburgel** [comment]
<< domainlgtm

File: Appraisals [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
   gem 'rack'
   gem 'rails-controller-testing'
-  gem 'activerecord-jdbcsqlite3-adapter', github: 'jruby/activerecord-jdbc-adapter', branch: 'rails-5', platforms: :jruby
 end
 
 appraise 'rails-5-1' do
   gem 'rails', '~> 5.1.0'
   gem 'rack'
   gem 'rails-controller-testing'
-  gem 'activerecord-jdbcsqlite3-adapter', github: 'jruby/activerecord-jdbc-adapter', branch: 'rails-5', platforms: :jruby
```
Comment (samandmoore) [inline Appraisals]: @smudge why is it that we can remove this from here? we don't need this on rails 5+?

Comment (samandmoore) [reply Appraisals]: oh, i'm guessing that it doesn't work on rails 5+, correct?

Comment (samandmoore) [reply Appraisals]: Nope, I'm still confused.

Comment (smudge) [reply Appraisals]: I've left a generic line in the`Gemfile`, which the other appraisals will inherit:

```
gem 'activerecord-jdbcsqlite3-adapter', platforms: :jruby
```

**Comment by samandmoore** [comment]
Ah. Yes yes. That’ll do it.

**Pull request merged**